### DISCUSSION TITLE
More spawner related features

### DISCUSSION THREAD

**Discussion by UsainSrht** [PurpurMC/Purpur#927] [Feature Requests]
setting for allowing every item to drop spawners
setting for disable breaking spawners without whitelisted item or not enough silk touch level
add support for <mob> placeholder

**Comment by RhythmicSys** [comment]
[The first 2 things you asked, already exist](https://purpurmc.org/docs/Configuration/silk-touch_1)
I am not sure what you mean by the third things though

**Reply by UsainSrht** [reply]
> [The first 2 things you asked, already exist](https://purpurmc.org/docs/Configuration/silk-touch_1) I am not sure what you mean by the third things though

its just a whitelist of items, how do I make it drop with every item in game?
and translate isn't possible on purpur's config rn it only shows <mob> as Skeleton or Slime

**Comment by Itsyourdriver** [comment]
> add support for placeholder


Placeholders as in something like placeholderapi?

**Comment by James-P-Bennett** [comment]
Expanding on this thread - https://github.com/PurpurMC/Purpur/discussions/1585
